cogs/info/serverinfo.py

```diff
@@ -29,29 +29,29 @@ async def serverinfo(self, ctx: commands.Context[YukiSuou]) -> None:
 
         guild_data = [
             f"- **Owner:** {guild.owner.mention if guild.owner else f'<@{guild.owner_id}>'} (`{guild.owner_id}`)",
-            f"- **ID: ** {guild.id}",
+            f'- **ID: ** {guild.id}',
             f"- **Created:** {discord.utils.format_dt(guild.created_at, 'D')} ({discord.utils.format_dt(guild.created_at, 'R')})",
         ]
-        embed.add_field(name="", value="\n".join(guild_data), inline=False)
+        embed.add_field(name='', value='\n'.join(guild_data), inline=False)
 
         valid_roles = [role.mention for role in guild.roles if role is not guild.default_role]
         valid_roles.reverse()
         emojis = [str(emoji) for emoji in guild.emojis]
         statistic_data = [
-            f"- **Members:** `{guild.member_count}`",
-            f"- **Channels:** `{len(guild.channels)}`",
+            f'- **Members:** `{guild.member_count}`',
+            f'- **Channels:** `{len(guild.channels)}`',
             (
-                f"- **Roles: ** {', '.join(valid_roles) if len(valid_roles) <= 3 else ', '.join(valid_roles[:3]) + f' + {len(valid_roles)-3} roles'}"
+                f"- **Roles: ** {', '.join(valid_roles) if len(valid_roles) <= 3 else ', '.join(valid_roles[:3]) + f' + {len(valid_roles)-3} roles'}"  # noqa: PLR2004
                 if valid_roles
                 else None
             ),
             (
-                f"- **Emojis: ** {' '.join(emojis) if len(emojis) <= 3 else ' '.join(emojis[:3]) + f' + {len(emojis)-3} emojis'} (`{len(guild.emojis)}/{guild.emoji_limit}`)"
+                f"- **Emojis: ** {' '.join(emojis) if len(emojis) <= 3 else ' '.join(emojis[:3]) + f' + {len(emojis)-3} emojis'} (`{len(guild.emojis)}/{guild.emoji_limit}`)"  # noqa: PLR2004
                 if valid_roles
                 else None
             ),
         ]
-        embed.add_field(name="Statistics", value="\n".join([data for data in statistic_data if data]), inline=False)
+        embed.add_field(name='Statistics', value='\n'.join([data for data in statistic_data if data]), inline=False)
 
```

Comment:
This feels readable and makes sense to do instead of whatever i was doing before.... twice..